rc/base/lint.kak

```diff
@@ -15,9 +15,15 @@ define-command lint -docstring 'Parse the current buffer with a linter' %{
             exit 1
         fi
 
+        basename="${kak_buffile##*/}"
```

Comment:
Do we need this basename ? I was hoping for a oneliner to get the extension.

Reply:
You thought of simply using `${kak_buffile##*.}`? So did I, but that doesn't work well with dotfiles (e.g. `.config`), is there a simpler way of handling that?